### PR TITLE
improve: Format component/property/method hints into paragraph format

### PR DESCRIPTION
**Pull request by michaelborn**
If line breaks are present in the "hint", format them into paragraphs for a much cleaner description.

i.e. take this documentation:

```
/**
 * Wraps the Vehicles API to provide higher-order methods for pulling vehicle data.
 *
 * If you find yourself manipulating vehicle data frequently and consistently, consider moving that vehicle transformation logic into this component.
 */
component{}
```

from this:
single-line component hint with no paragraph break

to this:

double-line component hint with paragraph break

### CODE REVIEW

File: strategy/api/resources/templates/class.cfm [hunk 1]
```diff
@@ -688,6 +688,28 @@ Class
 		}
 		*/
 
+		/**
```
Comment (lmajano) [inline strategy/api/resources/templates/class.cfm:691]: Can you run this via the cfformatter?

File: strategy/api/resources/templates/class.cfm [hunk 2]
```diff
@@ -688,6 +688,28 @@ Class
...
+		 * 
+		 * @cite https://cflib.org/udf/ParagraphFormat2
+		 * @param string      The string to format. (Required)
+		 * @return Returns a string. 
+		 * @author Ben Forta (ben@forta.com) 
+		 * @version 3, June 26, 2002 
+		 */
+		function formatLineBreaks( string val ){
+			// first make Windows style into Unix style
+			val = replace(val,chr(13)&chr(10),chr(10),"ALL");
```
Comment (lmajano) [inline strategy/api/resources/templates/class.cfm:702]: can you make sure you use the `arguments` scope.

Comment (michaelborn) [reply strategy/api/resources/templates/class.cfm:702]: Nice catch! Yep, I can do that. That's what I get for using CFLib functions from 2002...

File: strategy/api/resources/templates/class.cfm [hunk 3]
```diff
@@ -688,6 +688,28 @@ Class
 		}
 		*/
 
+		/**
+		 * Improved version of Ben Forta's "ParagraphFormat2" method on cflib.
```
Comment (lmajano) [inline strategy/api/resources/templates/class.cfm:692]: What's the difference in just using `paragraphFormat`? https://cfdocs.org/paragraphformat

**Comment by michaelborn** [comment]
@lmajano I looked this over again. I don't like the way I implemented this "feature", and I'd much rather move to alternate methods of improving the generated docs UI.

By the way: Lucee appears to condense multiple line breaks (i.e. `chr(10)`) inside a multiline comment into a single line break. This makes it impossible to format the following hint into two paragraphs without

```
/**
 * This is my multiline comment
 * which spans three lines
 * but is only a single paragraph.
 * 
 * This is a NEW paragraph denoted by two consecutive line breaks.
  */
```

TL;DR: Let's close this in favor of a Tailwind docs template.

**Comment by michaelborn** [comment]
Closing in favor of a Tailwind docs template.
